en/tutorials-and-examples/cms/authentication-plugin.rst

```diff
@@ -39,7 +39,7 @@ add the following::
     <?php
     namespace App\Model\Entity;
 
-    use Authentication\PasswordHasher\DefaultPasswordHasher; // Add this line
+    use Cake\Auth\DefaultPasswordHasher; // Add this line
```

Comment:
This will use the deprecated password hasher from the AuthComponent, not the one from the plugin.

Reply:
thanks for clarifying